lib/boundary.dart

```diff
@@ -170,6 +170,7 @@ class _Internal extends StatelessWidget {
     return _InheritedBoundary(
       element: element,
       child: Stack(
+        fit: StackFit.passthrough,
```

Comment:
Could you add a test for it?

The case you case in the issue combined with a `tester.getSize`/`tester.getCenter` should be enough

Reply:
Sure

Reply:
Done. I ended up grabbing the child constraints and comparing directly.

Also tested the negation, removing the `fit: StackFit.passthrough` from the library. It fails as expected.